gist/gist

```diff
@@ -0,0 +1,812 @@
+#!/usr/bin/env bash
+#
+# Author: Hsieh Chin Fan (typebrook) <typebrook@gmail.com>
```

Comment:
Can we add some of this to this [file](https://github.com/alexanderepstein/Bash-Snippets/blob/master/bash-snippets.1.ronn) and then use this [tool](https://github.com/rtomayko/ronn ) to generate this [file](https://github.com/alexanderepstein/Bash-Snippets/blob/master/bash-snippets.1)

Reply:
Done in f9e14ca